test/extensions/filters/http/grpc_json_transcoder/json_transcoder_filter_test.cc

```diff
@@ -766,6 +774,20 @@ class GrpcJsonTranscoderFilterConvertGrpcStatusTest : public GrpcJsonTranscoderF
   }
 };
 
+class GrpcJsonTranscoderFilterConvertGreaterMaximumKnownGrpcStatusTest
+    : public GrpcJsonTranscoderFilterConvertGrpcStatusTestBase {
+public:
+  GrpcJsonTranscoderFilterConvertGreaterMaximumKnownGrpcStatusTest()
+      : GrpcJsonTranscoderFilterConvertGrpcStatusTestBase(makeProtoConfig()) {}
+
+private:
```

Comment:
Isn't this same as `GrpcJsonTranscoderFilterTest` because convert_grpc_status is default false?

Reply:
No, I think setting up that is in `GrpcJsonTranscoderBase.Setup()` is needed. So there must be class that is inherited from `GrpcJsonTranscoderFilterStatusBase` and `convert_grpc_status` is true. But, there is no class to satisfy these requirements. So I changed class structure. @lizan

Reply:
well the makeProtoConfig is returning same config as bookstoreProtoConfig(), so this class does literally nothing...

Reply:
actually the `Setup` is not doing `Setup` can we change that to a different name and just explicitly call that? It can end up in `GrpcJsonTranscoderFilterTest`